### DISCUSSION TITLE
Search through notebook

### DISCUSSION THREAD

**Discussion by brainbow1** [zim-desktop-wiki/zim-desktop-wiki#2231] [Q&A] [answered]
Hello,
in my beloved zim wiki, v0.73.5, in the menu bar, there is a point under "Search", which is called in german "Notizbuch durchsuchen" -> "search through notebook".
I just found that function and find it to be highly useful as I have to search through my notebook after keywords quite often.. 

Unfortunately I couldnt find that menu entry within the keybindings-menu.
Does anybody have an idea how I can set that properly, lets say to "Ctrl+F", as a shortcut for that notebook-wide keywordsearch?

I appreciate ur help..
kind regards

**Comment by introt** [comment, marked as answer]
Hi!

```
#. Menu item
#: zim/gui/uiactions.py:276
msgid "_Search..."
msgstr "Notizbuch durchsuchen..."
```

You seem to be referring to the "global" search, which should be accessible by Shift + Ctrl + F by default. It's called `<Actions>/UIActions/show_search` in the keybindings-menu.

Hope this helps!

**Reply by brainbow1** [reply]
thank u very much for the quick reply!
When I press Shift+Ctrl+F, the "find and replace" dialog appears.... which is nice but not really the global search... 
I also cant find the global search written in the preferences-menu where they keybindings are set :-(
any ideas on this?

**Reply by introt** [reply]
Sorry, no idea - works as expected on my system (and Ctrl + H opens the find/replace dialog). Maybe try resetting the keybindings by moving the `~/.config/zim/accelmap` file somewhere else (to serve as a backup) and try restarting Zim?

Feel free to post your `accelmap` here and I'll have a look - mine does include `; (gtk_accel_path "<Actions>/UIActions/show_search" "<Primary><Shift>f")` as it should.

**Reply by brainbow1** [reply]
ahh I love you, it works now :-)
I was just too stupid to realize that the "show search" point under "UI Actions" is the point that I want to connect the respective keybinding to....
thanks a lot, saved my life :-)
